src/Authentication/Authentication/Common/Output.cs

```diff
@@ -0,0 +1,17 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph.PowerShell.Authentication.Common
+{
+    using System;
+
+    public class Output : IOutput
```

Comment:
This class should be under `models` directory for consistency - https://github.com/microsoftgraph/msgraph-sdk-powershell/tree/dev/src/Authentication/Authentication/Models.

Also, we should give this class a different name since Output is too generic and can be confused with [OutputType.cs](https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/dev/src/Authentication/Authentication/Models/OutputType.cs). Something like `PSOutputStream` or `OutputStream` would be ideal.